### PR TITLE
Python types and fixes

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add more address types, fix serialization, return accounts from get_accounts(), fix chain type

## Links to any relevant issues

#42 

## Type of change

- Bug fix (a non-breaking change which fixes an issue)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How the change has been tested

Running tests and examples

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/python/CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +43,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Account::prepare_send_native_tokens()` now accepts `SendNativeTokensParams`;
 - `Account::prepare_send_nft()` now accepts `SendNftParams`;
 - `Account::send_outputs()` now accepts `Output`;
+- `Account::{generate_ed25519_addresses(), addresses()}` return now `AccountAddress`;
```
Comment (thibault-martinez) [inline bindings/python/CHANGELOG.md]:
```suggestion
- `Account::{generate_ed25519_addresses(), addresses()}` now return `AccountAddress`;
```

File: bindings/python/iota_sdk/wallet/wallet.py [hunk 2]
```diff
@@ -153,22 +154,6 @@ def restore_backup(self, source: str, password: str):
             }
         )
 
-    def generate_mnemonic(self) -> str:
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/wallet/wallet.py:156]: Changelog?

Comment (Thoralf-M) [reply bindings/python/iota_sdk/wallet/wallet.py:156]: Added

File: bindings/python/CHANGELOG.md [hunk 3]
```diff
@@ -42,6 +43,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Account::prepare_send_native_tokens()` now accepts `SendNativeTokensParams`;
 - `Account::prepare_send_nft()` now accepts `SendNftParams`;
 - `Account::send_outputs()` now accepts `Output`;
+- `Account::{generate_ed25519_addresses(), addresses()}` now return `AccountAddress`;
+- `Account::addresses_with_unspent_outputs()` returns now `AddressWithUnspentOutputs`;
```
Comment (thibault-martinez) [inline bindings/python/CHANGELOG.md]:
```suggestion
- `Account::addresses_with_unspent_outputs()` now returns `AddressWithUnspentOutputs`;
```
And the others

Comment (Thoralf-M) [reply bindings/python/CHANGELOG.md]: Updated

File: bindings/python/CHANGELOG.md [hunk 4]
```diff
@@ -42,6 +43,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- `Wallet::get_accounts()` now returns `List[Account]`;
+- `OutputData.chain` is now `Optional[Bip44]`;
+
+### Fixed
+
+- Serialization for wallet methods;
+
+### Removed
+
+- `Wallet::{generate_mnemonic(), verify_mnemonic()}` since they're available from `Utils`;
```
Comment (thibault-martinez) [inline bindings/python/CHANGELOG.md:54]: I'm super annoying now but please put removed before fixed

**Review by thibault-martinez — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**